src/components/ListPane/ListPane.tsx

```diff
@@ -148,7 +152,9 @@ const List = () => {
                 backgroundColor: 'transparent',
                 ...(listParams.sortOrder === 'DESC' && { color: theme.colors.primary_5 }),
               }}
-              // onClick={() => setListParams({ ...listParams, sortOrder: 'DESC' })}
+              onClick={() =>
+                setListParams({ sortOrder: listParams.sortOrder === 'ASC' ? 'DESC' : 'ASC' })
```

Comment:
worth putting 'ASC' and 'DESC' into enum? used a fair bit

Reply:
Nvmd i def do...will use it here!